nc/source/physics/PhysicsSystem.h

```diff
@@ -18,6 +19,10 @@ namespace nc::physics
         public:
             PhysicsSystem(graphics::Graphics* graphics);
 
+            void ClearState();
+
```

Comment:
Lest you want to catch a case, best to delete that white space.

Reply:
lol! Trifle not with unseemly codebases.